app/index.html

```diff
@@ -117,6 +117,22 @@
                           </label>
                         </li>
 
+
+                        <li>
+                          <label
+                            class="toggle-switch light"
+                            for="toggle-dark-theme"
+                            title="Enable dark theme">
+                            <input
+                              type="checkbox"
+                              name="theme"
+                              id="toggle-dark-thme"
+                              ng-checked="settings.darkThemeEnabled"
```

Comment:
suggestion: we can skip maintaining the dark theme toggle state and just wrap your dark theme css styles with the `prefer-color-cheme` media query https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme. That way, it will change depending on if the user has their OS set to a dark theme.

Reply:
Good idea, I'll do it to set the default value of `settings.darkThemeEnabled`.  
I'll leave the toggle here, because some mails are unreadable with the dark mode.